volunteers.html

```diff
@@ -61,147 +63,377 @@
 		<div class="container team">
 
 		  <div class="row">
-        <h1>Instructors</h1>
+        <h1>Volunteers -- you made this happen!</h1>
```

Comment:
The bios are listed in sort of "order of appearance" here in one big grid list with everyone. I thought about alphabetizing, but that didn't seem right when it came out with either first or last name. I thought the whole order of appearance thing went well with the origin story in Marvel's bio. I figured that the order of appearance thing would also be easier to maintain with static html to just add newcomers to the end. It doesn't limit us from bumping people up and down the list (like Shannon for example who came in late but took on a big role.) It's just a lot of html to shuffle when we want to do that.